packages/mui-base/src/SelectUnstyled/SelectUnstyled.test.tsx

```diff
@@ -0,0 +1,188 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import SelectUnstyled, { selectUnstyledClasses } from '@mui/base/SelectUnstyled';
+import OptionUnstyled from '@mui/base/OptionUnstyled';
+import OptionGroupUnstyled from '@mui/base/OptionGroupUnstyled';
+import {
+  createMount,
+  createRenderer,
+  describeConformanceUnstyled,
+  fireEvent,
+  userEvent,
+  act,
+} from 'test/utils';
+
+describe('SelectUnstyled', () => {
+  const mount = createMount();
+  const { render } = createRenderer();
+
+  const componentToTest = (
+    <SelectUnstyled defaultListboxOpen>
+      <OptionGroupUnstyled label="Group">
+        <OptionUnstyled value={1}>1</OptionUnstyled>
+      </OptionGroupUnstyled>
+    </SelectUnstyled>
+  );
+
+  describeConformanceUnstyled(componentToTest, () => ({
+    inheritComponent: 'button',
+    render,
+    mount,
+    refInstanceof: window.HTMLButtonElement,
+    testComponentPropWith: 'span',
+    muiName: 'MuiSelectUnstyled',
+    slots: {
+      root: {
+        expectedClassName: selectUnstyledClasses.root,
+      },
+      listbox: {
+        expectedClassName: selectUnstyledClasses.listbox,
+        testWithElement: 'ul',
+      },
+      popper: {
+        expectedClassName: selectUnstyledClasses.popper,
+        testWithElement: null,
+      },
+    },
+  }));
+
+  describe('keyboard navigation', () => {
+    ['Enter', ' ', 'ArrowDown', 'ArrowUp'].forEach((key) => {
+      it(`opens the dropdown when the "${key}" key is pressed on the button`, () => {
+        // can't use the default native `button` as it doesn't treat enter or space press as a click
+        const { getByRole } = render(<SelectUnstyled components={{ Root: 'div' }} />);
+        const button = getByRole('button');
+        act(() => {
+          button.focus();
+          userEvent.keyPress(button, { key: 'Enter' });
```

Comment:
You are always sending here 'Enter', it should be key :)

Reply:
🤦‍♂️